packages/trader/src/Modules/Trading/Components/Form/TradeParams/Turbos/barriers-list.jsx

```diff
@@ -0,0 +1,65 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const BarriersItem = ({ active_item_class_name, base_class_name, id, is_active, hover_item_class_name, value }) => {
```

Comment:
BarriersList and BarriersItem are temporary names (as well as there position), I'll replace there names to more universal ones and remove to components package.

Reply:
Open for name suggestions

Reply:
@kate-deriv BarriersListItem, and the file name - barriers-list-item.jsx

Reply:
Then I'll integrate it in <ul> component